parastell/parastell.py

```diff
@@ -488,6 +497,15 @@ def export_cad_to_dagmc(
             max_mesh_size=max_mesh_size,
             imprint=False,
         )
+        if self.use_pydagmc:
+            self.magnet_model = export_path
+
+    def merge_magnet_and_ivb_dagmc_models(self):
+        magnet_mbc = core.Core()
+        magnet_mbc.load_file(str(self.magnet_model))
+        self.merged_model = merge_dagmc_files(
+            [self.invessel_build.dag_model.mb, magnet_mbc]
+        )
```

Comment:
I'm not sure there's much of a use-case for this to be a user-called function, since we know if a merged DAGMC model needs to be created based on whether an IVB and magnet set are created, and if PyDAGMC is being used. I think we should automate this.